restaurant/src/app/admin/view-sold-foods/view-sold-foods.component.html

```diff
@@ -0,0 +1,19 @@
+<h4 class="text-center"> Enter the dates to find the count </h4>
```

Comment:
`Enter the dates to find the count` change to `Enter the start date and end date to get the food order summary` or any other readable one